02_knight_and_princess/tamara_gadieva/graph_func.cpp

```diff
@@ -0,0 +1,43 @@
+#include "graph_generate.hpp"
+
+void Graph::add_vertex() {
+  vertices_.push_back(Vertex(get_new_vertex_id()));
```

Comment:
Читай `Common Mistakes`
```suggestion
  vertices_.emplace_back(get_new_vertex_id());
```